tests/foreman/cli/test_gpgkey.py

```diff
@@ -308,103 +293,83 @@ def test_negative_delete_1(self):
         @status: manual
         """
 
-    @stubbed()
-    @run_only_on('sat')
-    def test_negative_delete_2(self):
-        """@test: Create gpg key with valid name and valid gpg key text via
-        cut and paste/string then fail to delete it
-
-        @feature: GPG Keys
-
-        @assert: gpg key is not deleted
-
-        @status: manual
-        """
-
     # Positive Update
 
-    @stubbed()
     @run_only_on('sat')
-    def test_positive_update_1(self):
+    @tier1
+    def test_pos_update_name(self):
         """@test: Create gpg key with valid name and valid gpg key via file
         import then update its name
 
         @feature: GPG Keys
 
         @assert: gpg key is updated
-
-        @status: manual
         """
+        gpg_key = make_gpg_key({'organization-id': self.org['id']})
+        for new_name in valid_data_list():
+            with self.subTest(new_name):
+                GPGKey.update({
+                    'name': gpg_key['name'],
+                    'new-name': new_name,
+                    'organization-id': self.org['id'],
+                })
+                gpg_key = GPGKey.info({
+                    'name': new_name,
+                    'organization-id': self.org['id'],
+                })
 
-    @stubbed()
     @run_only_on('sat')
-    def test_positive_update_2(self):
+    @tier1
+    def test_pos_update_key(self):
         """@test: Create gpg key with valid name and valid gpg key via file
         import then update its gpg key file
 
         @feature: GPG Keys
 
         @assert: gpg key is updated
-
-        @status: manual
-        """
-
-    @stubbed()
-    @run_only_on('sat')
-    def test_positive_update_3(self):
-        """@test: Create gpg key with valid name and valid gpg key text via
-        cut and paste/string then update its name
```

Comment:
Deleted such stubs as we can't copypaste key via CLI. CLI only supports attaching key file:
```
Usage:
    hammer gpg create [OPTIONS]

Options:
 --key GPG_KEY_FILE                      GPG Key file
 --name NAME                             identifier of the gpg key
 --organization ORGANIZATION_NAME        Organization name to search by
 --organization-id ORGANIZATION_ID       organization ID
 --organization-label ORGANIZATION_LABEL Organization label to search by
```